### DISCUSSION TITLE
line and arc are one tool

### DISCUSSION THREAD

**Discussion by 4a3** [FreeCAD/FreeCAD#11897] [Code contribution discussions]
It would be more convenient if it were possible to bend a straight line and straighten the arch.:
https://github.com/FreeCAD/FreeCAD/assets/829542/5d7ab593-ab69-412a-a53b-ceb878818cd8

**Comment by pierreporte** [comment]
Interesting. You should create a feature request: https://github.com/FreeCAD/FreeCAD/issues/new/choose

**Comment by 4a3** [comment]
in "backport" or "problem"?

**Comment by pierreporte** [comment]
No, in “report a problem”.

**Comment by 4a3** [comment]
thank you
